engineyard/ey-cookbooks-stable-v6
# [FB-830] Enable memcached recipe with EY_MEMCACHED_ENABLED

#### Description of your patch
Enable memcached recipe with EY_MEMCACHED_ENABLED environment variable
https://t3.fogbugz.com/f/cases/830

#### Recommended Release Notes
Enable memcached recipe with EY_MEMCACHED_ENABLED environment variable

#### Estimated risk
Low

#### Components involved
Memcached

#### Description of testing done
See QA instructions

#### QA Instructions
Add EY_MEMCACHED_ENABLED environment variable using any value (e.g. 1)
Boot a utility instance named memcached
Check that memcached is running.

```
ps -ef | grep memcached
systemctl status memcached
```
